_posts/2016-01-05-aplicando-gqm-na-vida.markdown

```diff
@@ -0,0 +1,159 @@
+várias pessoas extremamente experientes. E uma destas pessoas detectou este meu
+amado problema. E para atacar este problema ele propôs que eu estruturasse
+meus objetivos, e tivesse uma forma clara de mensurar meu progresso e se já
+cumpri a profundidade inicial pretendida.
+
+Como é de se esperar, existem diversos modelos sistematizadas para se fazer
+isto, e eu não passei ileso de sugestões, talvez a mais conhecida sugestão seja
+o [OKR](https://en.wikipedia.org/wiki/OKR) adotada por diversas empresas como
+*Google* por exemplo. Uma outra sugestão foi o
+[GQM](https://en.wikipedia.org/wiki/GQM) (Goal, Question, Metric) que foi
+um modelo criado especificamente para software mas pode facilmente adaptado para
+ajudá-lo no dia-a-dia.
+
+Acabei, por fim, optando pelo [GQM](https://en.wikipedia.org/wiki/GQM) por um
+motivo: Embora não tenho um conhecimento profundamente em
+[OKR](https://en.wikipedia.org/wiki/OKR), eu já vi ser aplicado - ou tentando
+ser aplicado - em alguns lugares, e o [GQM](https://en.wikipedia.org/wiki/GQM)
+era novidade.
+
+## Eis aqui a minha instancia de GQM
+
+Não vou explicar o que é exatamente o [GQM](https://en.wikipedia.org/wiki/GQM),
+o link que eu estou colocando dá uma introdução muito melhor do que eu poderia
+fazer. O único contra, é ainda não haver um Wiki em português sobre o assunto.
+Injustiça que pode ser corrigido por um de nós.
+
+O [GQM](https://en.wikipedia.org/wiki/GQM) pede que você faça as seguintes
+definições:
+
+### Objeto de medição (Object of mensurement)
+
+Vou escolher apenas um Objeto: **Me tornar um Cientista da Computação decente**.
+
+Existe aqui algo que possa ser entendido como uma incompatibilidade entre o que
+o [GQM](https://en.wikipedia.org/wiki/GQM) entende sobre o que é objeto de
+medição, e o que eu vou considerar como objeto de medição. Algo que é esperado
+dado a natureza sobre a qual o sistema foi criado (desenvolvimento de software).
+
+O sistema diz que **objeto de mesuração** é uma das três coisas a seguir:
+Produtos, Processos ou Recursos.
+
+Porém se eu esticar a definição um pouco, posso encaixar meu conhecimento como
+um Recurso. Embora isto pareça um pouco contra intuitivo inicialmente, estou
+mais confortável com isto agora (tipo: aceita que dói menos).
+
+### Modelo de medição
+
+1. Objetivos (GOAL) que darão suporte para que o objeto selecionado a ser
+   construído.
+   Vou selecionar apenas um objetivo:
+   * Concluir o livro The art of computer programming Vol. I.
+2. Perguntas (QUESTIONS) que devem ser respondidas para se medir se o objetivo
+   será cumprido, tentando verificar se caminho para a construção do objeto não
+   foi perdido.
+   1. Sou capaz de explicar e utilizar os fundamentos matemáticos para analise
+      de algoritimos e a máquina MIX?
+   2. Sou capaz de explicar os fundamentos de técnica de programação?
+   3. Sou capas de explicar e utilizar as estruturas de dados do capitulo 2 do
+      livro TAOCP?
```

Comment:
Se vc for usar uma sigla, use primeiro logo na sequência do texto que ela representa, ou seja, lá qdo vc citou o livro a primeira vez